# HG changeset patch
# User Keir Fraser <keir.fraser@citrix.com>
# Date 1219920196 -3600
# Node ID 83fb5b3e55e12e2b94d34eb168c3c5c30e75d089
# Parent 05e59668aa77d9566b22796768f7fe6a6d89b915
Use 'gcc -v 2>&1 | tail -1' to extract gcc version.

Signed-off-by: Keir Fraser <keir.fraser@citrix.com>
xen-unstable changeset:   18380:6c6bda7f09cde36fa875941d7202e77620fdc687
xen-unstable date:        Wed Aug 27 11:47:02 2008 +0100

--- a/xen/Makefile	Thu Aug 28 11:41:56 2008 +0100
+++ b/xen/Makefile	Thu Aug 28 11:43:16 2008 +0100
@@ -87,7 +87,7 @@ include/xen/compile.h: include/xen/compi
 	    -e 's/@@whoami@@/$(XEN_WHOAMI)/g' \
 	    -e 's/@@domain@@/$(XEN_DOMAIN)/g' \
 	    -e 's/@@hostname@@/$(shell hostname)/g' \
-	    -e 's!@@compiler@@!$(shell $(CC) $(CFLAGS) -v 2>&1 | grep -i "gcc.*version")!g' \
+	    -e 's!@@compiler@@!$(shell $(CC) $(CFLAGS) -v 2>&1 | tail -1)!g' \
 	    -e 's/@@version@@/$(XEN_VERSION)/g' \
 	    -e 's/@@subversion@@/$(XEN_SUBVERSION)/g' \
 	    -e 's/@@extraversion@@/$(XEN_EXTRAVERSION)/g' \
